token-service/src/main/java/org/p2p/token/service/repository/price/TokenPriceDatabaseRepository.kt

```diff
@@ -36,3 +36,7 @@ internal class TokenPriceDatabaseRepository(
             .map(converter::fromEntity)
     }
 }
+
+internal fun String.toPriceColumnKey(networkChainName: String): String {
```

Comment:
and put it in the Entity too
that's why we are extracting this logic to a method - to remove errors if it's changed